plugins/woocommerce/client/blocks/assets/js/atomic/utils/render-parent-block.tsx

```
@@ -216,54 +216,58 @@ const renderInnerBlocks = ( {
+							)
+						}
+					</InnerBlockComponent>
+				</InnerBlockComponentWrapper>
+			</BlockErrorBoundary>
+		);
+
+		// Temporary return until the Cart block is also updated
+		return isCheckoutBlock ? (
+			blockContent
+		) : (
+			<Suspense fallback={ null }>{ blockContent }</Suspense>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)